contributing/code/standards.rst

```diff
@@ -1,19 +1,32 @@
 Coding Standards
 ================
 
-When contributing code to Symfony, you must follow its coding standards. To
-make a long story short, here is the golden rule: **Imitate the existing
-Symfony code**. Most open-source Bundles and libraries used by Symfony also
-follow the same guidelines, and you should too.
+Symfony code is contributed by thousands of developers around the world. To make
+every piece of code look and feel familiar, Symfony defines some coding standards
+that all contributions must follow.
 
-Remember that the main advantage of standards is that every piece of code
-looks and feels familiar, it's not about this or that being more readable.
+These Symfony coding standards are based on the `PSR-0`_, `PSR-1`_, `PSR-2`_ and
+`PSR-4`_ standards, so you may already know most of them.
 
-Symfony follows the standards defined in the `PSR-0`_, `PSR-1`_, `PSR-2`_ and `PSR-4`_
-documents.
+Making your Code Follow the Coding Standards
+--------------------------------------------
 
-Since a picture - or some code - is worth a thousand words, here's a short
-example containing most features described below:
+Instead of reviewing your code manually, Symfony makes it simple to ensure that
+your contributed code matches the expected code syntax. First, install the
+`PHP-CS-Fixer tool`_ and then, run this command to fix any problem:
+
+.. code-block:: terminal
+
+    $ php php-cs-fixer.phar fix /path/to/project --rules=@Symfony
```

Comment:
providing a path is also bad, as it will override path configured in `.php_cs.dist` file, leading to fixing even **ignored** files., command shall be just `php php-cs-fixer.phar fix -v` executed from root project dir